qds_sdk/qubole.py

```diff
@@ -43,6 +46,9 @@ def configure(cls, api_token,
         cls._auth = QuboleAuth(api_token)
         cls.api_token = api_token
         cls.base_url = api_url.rstrip('/') + '/' + version
+        if poll_interval < Qubole.MIN_POLL_INTERVAL:
+            sys.stderr.write("Poll interval cannot be less than %s seconds.\n" % Qubole.MIN_POLL_INTERVAL)
+            sys.exit(2)
```

Comment:
Don't exit with error, if you do, it will break for people who are currently using a very low value.
Instead if the poll_interval is less than Qubole.MIN_POLL_INTERVAL, set the poll interval to MIN_POLL_INTERVAL and display a warning log message that we have done so.

In general, while making changes to the SDK, be very careful about not changing the behavior in ways that will break existing workflows.